src/main/java/org/galatea/starter/domain/AlphaVantageMetaData.java

```diff
@@ -1,17 +1,21 @@
 package org.galatea.starter.domain;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import javax.persistence.Embeddable;
+import lombok.Data;
 
+@Data
+@Embeddable
 public class AlphaVantageMetaData {
-    @JsonProperty("1. Information")
-    private String information;
-    @JsonProperty("2. Symbol")
-    private String symbol;
-    @JsonProperty("3. Last Refreshed")
-    private String lastRefreshed;
-    @JsonProperty("4. Output Size")
-    private String outputSize;
-    @JsonProperty("5. Time Zone")
-    private String timeZone;
+  @JsonProperty("1. Information")
+  private String information;
```

Comment:
Could you add some spacing in-between these so it easier to read?

Reply:
done